src/docs/release-notes/R23/FullNotes.mdx

```diff
@@ -0,0 +1,33 @@
+### Documentation Site
+
+-   Added design guidelines for [the Tables design pattern](/patterns/tables).
+-   Added a color inspector to [the color palette page](/style/color). Now you may click on a color patch and view the color code in other color spaces and its color contrast ratios against other BLUI colors.
+
+### Angular
+
+-   Updated packages to support Angular Material v13
+-   Dropped support for Ionic
+
+### React
+
+-   Updated packages to support MUI v5
+
+### React Native
+
+-   Dropped support for Expo
+
+### Icons
+
+-   Added 5 new icons.
+
+### Design Resources
+
+-   Added more card components to the [Component Sticker Sheet](https://www.figma.com/community/file/1024360297793425107) on Figma.
+-   Added more customization options for the Figma plugin [Variant Switcher](https://www.figma.com/community/plugin/971482182464094790).
+-   Added a [video tutorial](https://www.youtube.com/watch?v=JBiuas4_Occ) for Variant Switcher.
+
+### Miscellaneous
+
+-   Bug fixes & minor updates
+
+### Package Summary
```

Comment:
Is this Package Summary supposed to be here? There's currently no notes below it.

Reply:
This PR is just for design notes; leaving this section here to remind devs to fill it in.